salt/engines/webhook.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+'''
+Send events from webhook api
+'''
+from __future__ import absolute_import
+
+# import tornado library
+import tornado.httpserver
+import tornado.ioloop
+import tornado.web
+
+# import salt libs
+import salt.utils.event
+
+
+def start(address=None, port=5000, ssl_crt=None, ssl_key=None):
+    '''
+    Api to listen for webhooks to send to the reactor.
+
+    Implement the webhook behavior in an engine.
+    :py:class:`rest_cherrypy Webhook docs <salt.netapi.rest_cherrypy.app.Webhook>`
+
+    Unlike the rest_cherrypy Webhook, this is only an unauthenticated webhook
```

Comment:
Suggestion: use a warning to visually call this out:
```
.. warning:: Unauthenticated endpoint

    Unlike the rest_cherrypy Webhook, this is...
```

Also suggest removing the note about minion-only. Because it's pretty common to disable auth for webhooks and this is easier to set up than rest_cherrypy I can def see this being commonly used on a master.